.github/workflows/pm.yml

```diff
@@ -7,14 +7,23 @@ on:
   pull_request:
     types:
       - opened
+      - ready_for_review
 
 jobs:
-  add-to-project:
+  automate:
     if: github.event.pull_request.head.repo.fork == false
     name: Add issue to project
     runs-on: ubuntu-latest
+    timeout-minutes: 5
     steps:
-      - uses: actions/add-to-project@v0.5.0
+      - uses: ory-corp/planning-automation-action@main
         with:
-          project-url: https://github.com/orgs/ory-corp/projects/5
-          github-token: ${{ secrets.ORY_BOT_PAT }}
+          project: 5
+          organization: ory-corp
+          token: ${{ secrets.ORY_BOT_PAT }}
+          todo_label: "Needs Triage"
```

Comment:
nit: Is this a new field? Shouldn't it be `camelCase` as well?

Reply:
good point, I'll update related PR on ory-corp/planning-automation-action and this.